db/schema.rb

```diff
@@ -32,9 +32,11 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "role"
+    t.integer  "user_id"
```

Comment:
I ran the above and it seems to have done the trick.

Just to explain what happened - when I tried to view a user profile I got an error saying that it could not find an ID for the user. So like an idiot I tried to create a user ID (even though in the back of my mind I knew that the table already had an ID). Silly mistake and I won't make it again. Kind of realised I made a mistake as soon as I did it.